upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -90,6 +90,9 @@ func (tf *TemplateFunctions) WellKnownServiceIP(id int) (net.IP, error) {
 func (tf *TemplateFunctions) AddTo(dest template.FuncMap) {
 	dest["SharedVPC"] = tf.SharedVPC
 
+	// Remember that we may be on a different arch from the target.  Hard-code for now.
+	dest["Arch"] = func() string { return "amd64" }
```

Comment:
We should have an issue for multi-arch.  But right now I'm just laying the groundwork where it costs us nothing.